typespec-extension/src/code-model-builder.ts

```diff
@@ -829,67 +801,75 @@ export class CodeModelBuilder {
 
     // host
     clientContext.hostParameters.forEach((it) => codeModelOperation.addParameter(it));
-    // parameters
-    op.parameters.parameters.map((it) => this.processParameter(codeModelOperation, it, clientContext));
-    // "accept" header
-    this.addAcceptHeaderParameter(codeModelOperation, op.responses);
-    // body
-    if (op.parameters.body) {
-      if (op.parameters.body.property) {
-        if (!isVoidType(op.parameters.body.property.type)) {
-          this.processParameterBody(codeModelOperation, op, op.parameters.body.property);
+    // path/query/header parameters
+    for (const param of httpOperation.parameters) {
+      // if it's paged operation with request body, skip content-type header added by TCGC, as next link call should not have content type header
+      if (
+        (sdkMethod.kind === "paging" || sdkMethod.kind === "lropaging") &&
+        httpOperation.bodyParam &&
+        param.kind === "header"
+      ) {
+        if (param.serializedName.toLocaleLowerCase() === "content-type") {
```

Comment:
there is a CONTENT_TYPE_KEY defined somewhere

Reply:
Got it, I saw it in operation-utils.ts, but the `CONTENT_TYPE_KEY` is not exported, I will export the constant and use it in code-model-builder.ts